spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1369,16 +1363,23 @@ def _do_load(self):
         # algorithms needed for loading the binaries to the SpiNNaker machine
         optional_algorithms.append("FrontEndCommonLoadExecutableImages")
 
+        # Get the executable targets
+        optional_algorithms.append("FrontEndCommonGraphBinaryGatherer")
```

Comment:
if done before loading, then your virutal board needs to find them. by being done at loading the virtual board can work, and as the executables are only found jsut before they are loaded, they might as well be in loading. 

i dont mind moving them about, we just put them at the first poitn where optional algorithms was built.